extensions/adobe/experience/lifecycle/lifecycle.schema.json

```diff
@@ -0,0 +1,97 @@
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/lifecycle",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Lifecycle Fields",
+  "type": "object",
+  "meta:extensible": false,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "description": "A set of meta-data fields that describe lifecycle events related to AJO entities.",
+  "definitions": {
+    "lifecycle": {
+      "properties": {
+        "https://ns.adobe.com/experience/lifecycle/status": {
+          "title": "Status",
+          "type": "string",
+          "description": "The lifecycle status of the journey step (e.g., draft, live, completed)."
+        },
+        "https://ns.adobe.com/experience/lifecycle/statusCode": {
+          "title": "Status Code",
+          "type": "string",
+          "description": "A machine-readable code representing the lifecycle status."
+        },
+        "https://ns.adobe.com/experience/lifecycle/processType": {
+          "title": "Process Type",
+          "type": "string",
+          "description": "The type of process being executed (e.g., publishing, stopping, delivering)."
+        },
+        "https://ns.adobe.com/experience/lifecycle/processStatus": {
+          "title": "Process Status",
+          "type": "string",
+          "description": "Status of the process (e.g., processing, processed, failed, completed)."
+        },
+        "https://ns.adobe.com/experience/lifecycle/sourceId": {
+          "title": "Source ID",
+          "type": "string",
+          "description": "Identifier for the AJO entity (e.g., journey ID, campaign ID)."
+        },
+        "https://ns.adobe.com/experience/lifecycle/sourceType": {
+          "title": "Source Type",
+          "type": "string",
+          "description": "Type of the AJO entity (e.g., campaign, journey)."
+        },
+        "https://ns.adobe.com/experience/lifecycle/sourceName": {
+          "title": "Source Name",
+          "type": "string",
+          "description": "Human-readable name of the AJO entity."
+        },
+        "https://ns.adobe.com/experience/lifecycle/isRecurrent": {
+          "title": "Is Recurrent",
+          "type": "boolean",
+          "description": "Indicates if this lifecycle event is part of a recurring AJO entity."
+        },
+        "https://ns.adobe.com/experience/lifecycle/batchInstanceId": {
+          "title": "Batch Instance ID",
+          "type": "string",
+          "description": "Unique identifier for the batch instance."
+        },
+        "https://ns.adobe.com/experience/lifecycle/recurrenceIndex": {
+          "title": "Recurrence Index",
+          "type": "integer",
+          "description": "Index value representing the recurrence index of a recurring AJO entity."
+        }
+      },
+      "required": [
```

Comment:
These should not be required. This field group will be included with other field groups on an experience event and not all events will be life cycle events. Therefore these can not be required.